raiden/raiden_service.py

```diff
@@ -153,12 +153,12 @@ def endpoint_registry_exception_handler(greenlet):
 
         if rpc_unreachable:
             log.fatal(
```

Comment:
`log.fatal` should be replaced with `log.exception` or `log.critical` (`fatal` log level is not available in structlog)